integration-tests/clipper_manager_tests.py

```diff
@@ -8,7 +8,7 @@
 from argparse import ArgumentParser
 cur_dir = os.path.dirname(os.path.abspath(__file__))
 sys.path.insert(0, os.path.abspath('%s/../' % cur_dir))
-import clipper_manager
+from clipper_admin import clipper_manager
```

Comment:
Ahh yeah good catch. This is a holdover from before the package structure existed.